modules/core/src/main/java/org/apache/ignite/internal/processors/query/QueryHistory.java

```diff
@@ -162,13 +162,13 @@ public long lastStartTime() {
      * @param updatedLink New link which should be set.
      * @return {@code true} If link has been updated.
      */
-    public boolean replaceLink(ConcurrentLinkedDeque8.Node<QueryHistoryMetrics> expLink,
-        ConcurrentLinkedDeque8.Node<QueryHistoryMetrics> updatedLink) {
+    public boolean replaceLink(ConcurrentLinkedDeque8.Node<QueryHistory> expLink,
+        ConcurrentLinkedDeque8.Node<QueryHistory> updatedLink) {
```

Comment:
Seems  the only `Node` class can be imported:  `import org.jsr166.ConcurrentLinkedDeque8.Node;`